libnd4j/buildnativeoperations.sh

```diff
@@ -56,6 +56,8 @@ VERBOSE="false"
 VERBOSE_ARG="VERBOSE=1"
 HELPER=
 NAME=
+SHARED_LIB_PREFIX="lib"
+SHARED_LIB_EXTENSION="so"
```

Comment:
I'm sure we can do something nicer with CMake instead of this here...

Reply:
Ha! Good point! :+1:

Reply:
It sounds like what we want is something like `make install` that copies everything in directories like `install_dir/bin`, `install_dir/include`, and `install_dir/lib`, by using the `CMAKE_INSTALL_PREFIX` variable. That's pretty standard for CMake and would allow us to create clean packages just for libnd4j out of that too.

Reply:
Yes, that will be great once we're done with API polishing and library  is usable as shared library by someone outside of the team. It would be awesome to refactor everything down to "single header include"

Reply:
It doesn't need to be a single header include or anything, just using the mechanism for `make install` for now to *copy* libraries instead shouldn't be a problem.

Reply:
But in case of windows, there's no install include/lib/etc dirs, we just need this shared libraries to be available. I.e. in the current PATH.

And that's exactly what we do now - POST-BUILD step in cmake does copy.

Reply:
That's what `CMAKE_INSTALL_PREFIX` is for, you've obviously never used it before. :)

Reply:
Nope, i hadn't